documentation/book/api/overview.adoc

```diff
@@ -1,9 +1,9 @@
-= Strimzi Kafka Bridge API Reference
+= Kafka Bridge API Reference
 
 
 [[_overview]]
 == Overview
-The Strimzi Kafka Bridge provides a REST API for integrating HTTP based client applications with a Kafka cluster. You can use the API to create and manage consumers and send and receive records over HTTP rather than the native Kafka protocol.
+The Kafka Bridge provides a REST API for integrating HTTP based client applications with a Kafka cluster. You can use the API to create and manage consumers and send and receive records over HTTP rather than the native Kafka protocol.
```

Comment:
I guess this needs to be also changed in the OpenAPI JSONs here: https://github.com/strimzi/strimzi-kafka-bridge/tree/main/src/main/resources?

Reply:
Yeah, I would change both `title` and `description` on both the openapi JSONs.

Reply:
Right. Thanks